.swiftlint.yml

```diff
@@ -1,3 +1,5 @@
 disabled_rules:
   - line_length
   - nesting
+identifier_name:
+  min_length: 1
```

Comment:
What was the motivation behind adding this change? Were we getting an error from something?

Reply:
Yeah, the tests on the CI were failing, so I had to make this change. The min length is 3 (default), but there were some variables in the tests that were of length 1 and SwiftLint didn't like that so the build failed.

Reply:
What were the variables out of interest? I'd rather fix those if we can

Reply:
The variables are located in the file `TABResourceLoader/Concrete types/Result.swift`, line 50 & 51:

```
case .success(let v): return v
case .failure(let e): throw e
```

The names`v` and `e` need to be updated so they are at least 3 or more characters in length,

Reply:
Yeah I think we should strive for best practices and not just disable rules to get round them ;) 

Can we change those variables to be something a bit more descriptive? I have no idea what v is even meant to stand for here!

Maybe `v` should actually be something like `success` or `model` and then `e` be `error`?

Reply:
Fixed it!